### PR TITLE
Update deps

### PR DESCRIPTION
**Pull request by alextargov**
**Description**

Changes proposed in this pull request:
- Update component dependencies 

**Related issue(s)**

- ...

**Pull Request status**

- [ ] Implementation
- [ ] Unit tests
- [ ] Integration tests
- [x] `chart/compass/values.yaml` is updated

### CODE REVIEW

**Review by desislavaa — COMMENTED** [review]
The E2E tests Dockerfile can also be updated with Go v16

**Review by dzahariev — COMMENTED** [review]
In addition:
Update schema migrator image to 3.13.5

All those changes are part of PR: https://github.com/kyma-incubator/compass/pull/1836
It was created only to validate proposed changes. 
If you prefer to not add those changes here, but to rebase the PR later, let me know, otherwise I will abandon my PR once the proposed changes are part of this one.

**Review by dzahariev — APPROVED** [review]

File: chart/compass/values.yaml [hunk 1]
```diff
@@ -71,21 +71,21 @@ global:
       version: "PR-1780"
```
Comment (dzahariev) [inline chart/compass/values.yaml:71]: Update also schema migrator image.

**Review by dzahariev — CHANGES_REQUESTED** [review]

File: components/operations-controller/Dockerfile [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # Build the manager binary
-FROM golang:1.15.7-alpine3.12 as builder
+FROM golang:1.16.3-alpine3.13 as builder
```
Comment (dzahariev) [inline components/operations-controller/Dockerfile:3]: Please add "-x" fo download (to get download feedback during build) on line 10:
```
RUN go mod download -x
```

**Review by dzahariev — APPROVED** [review]

**Pull request merged**